docs/deploy_local.md

```diff
@@ -51,6 +51,7 @@ export BOND_MEDIATOR_CONTRACT=${BondMediator}
 export BOND_FACTORY_CONTRACT=${BondFactory}
 export COLLATERAL_TOKENS_CONTRACT=${BitToken}
 export TREASURY_ADDRESS=${Treasury}
+export DAO_METADATA=${metadataString}
```

Comment:
issue: Variable consistency, use PascalCase for `MetadataString`.

_You've got caemCase and the others are all PascalCase